### ISSUE TITLE
Craft 3 update plans

### ISSUE THREAD

**Issue by echantigny** [michaelrog/Shortlist#26]
I'm wondering if there's any chance this plugin will get pushed to Craft 3 and Commerce 2 ?  I would like to use it to make Wishlists for my customers.

**Comment by jasonwilliamsau** [comment]
Is there an ETA for a C3 release yet?

**Comment by michaelrog** [comment]
Yes, a successor to _Shortlist_ is in the works. I hope to release it soon!

(If you need it urgently for a project and have some budget to help bump it up the priorities list, hit me up: support@topshelfcraft.com)

**Comment by j-greig** [comment]
Yasssss, excited to hear this news 👍

**Comment by tyssen** [comment]
Any more word on an ETA?

**Comment by michaelrog** [comment]
The next update on it will likely be the actual Beta release, which I hope will be very soon!

**Comment by glennjacobs** [comment]
Any ETA yet?
